tests/print_status/run.sh

```diff
@@ -50,6 +52,23 @@ function run() {
     check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
     # check sync unit print status
+    i=0
+    while [ $i -lt 3 ]
+    do
+        event_count=$(grep "receive binlog event with header" $WORK_DIR/worker1/log/dm-worker.log|grep -v relay|grep "XIDEvent"|wc -l)
```

Comment:
grep from _debug_ level log?

Reply:
yep, the log level in integration test is DEBUG

Reply:
OK. but notice the change of the log.